snmp/assets/configuration/spec.yaml

```diff
@@ -42,6 +42,14 @@ files:
         value:
           example: 10
           type: integer
+      - name: min_collection_interval
+        value:
+          example: 15
+          type: number
+        description: |
+          This changes the collection interval of the check. For more information, see:
```

Comment:
<!-- Suggestions propose improvements to the current subject. It’s important to be explicit and clear on what is being suggested and why it is an improvement. -->
**💬 suggestion**
Would rephrase it to 
```suggestion
          Minimum interval of time between two check runs, in seconds. For more information, see:
```

Reply:
This is the same description used for other integrations in integrations-core:

![image](https://user-images.githubusercontent.com/49917914/157427781-76f84f21-97dd-4736-9e9f-002aec14c7bb.png)

I think we should keep it for consistency. Or change it everywhere (but likely as a separate PR since not in scope of this PR).